### DISCUSSION TITLE
Welcome to System.Graphics Discussions!

### DISCUSSION THREAD

**Discussion by jonlipsky** [dotnet/System.Graphics#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thomiel** [comment]
Hi Jon! I'm Thomas, app developer with a private interest in looking under the hood of maui, especially extending it in a way that it can be used on linux OSes. 

Saw that there is already a gtk platform folder. What's the status? Could the recently added .NET6 workload of GtkSharp be helpful here?

Itm. I cannot build the current main branch. Any special preconditions? Would be helpful to have a "How to Build" section in the readme.

**Reply by jonlipsky** [reply]
Hi @thomiel,

I myself haven't looked at the GTK contribution yet.  At this point, I think it would be best characterized as a community contribution.  

With regards to building, I just open the Net 6 solution in Rider on my Mac (mainly) or Windows machine and build.  I haven't tried to build on Linux (which I'm assuming your trying to do).  

I do use the SkiaSharp implementation of Maui graphics on Linux quite a bit as part of some .net 6 ASP.net core services, but I myself no longer do any Linux desktop development.

Jon...

**Reply by thomiel** [reply]
Actually, I was trying with VS 2022 Preview on Window and made Progress after installing the .NET 5 SDK. But then, I was wondering why the default nuget.org package source was missing in the NuGet.config, so the four SharpDX.* subpackages couldn't be restored, which were needed for WPF. Wouldn't it be better to rely on nuget.org packages rather to refer to this bunch of home-made packages sources?

**Reply by thomiel** [reply]
I tried to build on my Mac Mini as well, but ran out of disk space and couldn't install the needed Android SDK. (It was just intended to be a build machine for iOS with only a 128 GB SSD, which I now regret badly.)

**Comment by jonlipsky** [comment]
@thomiel Let me look into the nuget issue.  As far as I knew, it should have be using nuget.org, so perhaps the Microsoft build team modified something along the line.
